### DISCUSSION TITLE
Can't access welcome page

### DISCUSSION THREAD

**Discussion by 13RTK** [logto-io/logto#2307] [Q&A] [answered]
I follow this [doc](https://docs.logto.io/docs/tutorials/get-started/) of the docker part, I successfully install the logto cli and seed the database via my local PostgreSQL database.
But after I pull the image(svhd ver.) and run it as a container, I can't access the welcome page, it has always been redirected to "https://your-domain-url/welcome".

This is my script for running the container:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/63441399/199460532-db749c82-13f6-444f-931a-539c1d0e6d9a.png">

This is the container's output:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/63441399/199459578-002ca9bd-f7f8-4919-adfa-6bbea98f0980.png">

This is request behavior:
![image](https://user-images.githubusercontent.com/63441399/199460015-c29bc32e-b295-4cab-b823-16913c6fa16f.png)

My pc is an m1 MacBook Air(ARM64), my macOS version is Monterey 12.6, and my terminal is zsh 5.8.1 (x86_64-apple-darwin21.0)

**Comment by gao-sun** [comment, marked as answer]
replace `your-domain-url` with a valid domain, otherwise you can leave the `ENDPOINT` environment variable empty, and open `http://localhost:3001` in your browser instead.

**Reply by 13RTK** [reply]
Thank you so much, I solved it by deleting the "ENDPOINT" environment variable.
